aiosmtpd/tests/test_starttls.py

```diff
@@ -67,6 +67,53 @@ def handle_STARTTLS(self, server, session, envelope):
         return False
 
 
+class EOFingHandler:
```

Comment:
This whole class, checked out from #211 (which itself adapted from #202) fixes the "intermittent coverage fail" problem.